packages/paste-core/components/table/src/Td.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {TdProps, TdPropTypes} from './types';
+
+const Td = React.forwardRef<HTMLTableCellElement, TdProps>(({alignment = 'left', width, ...props}, ref) => {
+  return (
+    <Box
+      {...safelySpreadBoxProps(props)}
+      ref={ref}
+      as="td"
+      fontSize="fontSize30"
+      lineHeight="lineHeight30"
+      paddingBottom="space40"
+      paddingLeft="space40"
+      paddingRight="space40"
+      paddingTop="space40"
+      position="relative"
+      textAlign={alignment}
+      verticalAlign="inherit"
+      width={width}
```

Comment:
I'm tempted to only have width set on the column header. Setting width on a cell seems odd, and limiting to headers focuses the size setting API to one area of a table. Thoughts?

Reply:
Yeah. I think thats a good idea. Seems weird to set the width in multiple areas.